### PR TITLE
Add oscar as author

### PR DESCRIPTION
**Pull request by oscarguindzberg**

### CODE REVIEW

File: rskj-core/src/main/java/co/rsk/util/peg/ReadBridgeState.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
+package co.rsk.util.peg;
```
Comment (bcodesido) [inline rskj-core/src/main/java/co/rsk/util/peg/ReadBridgeState.java]: This file is correctly committed?

File: rskj-core/src/main/resources/rskj.conf [hunk 2]
```diff
@@ -0,0 +1,326 @@
+peer.discovery = {
```
Comment (bcodesido) [inline rskj-core/src/main/resources/rskj.conf]: This file is correctly committed?

File: rskj-core/src/main/java/co/rsk/util/peg/ReadBridgeState.java (hunk 1 above)
Comment (oscarguindzberg) [inline rskj-core/src/main/java/co/rsk/util/peg/ReadBridgeState.java]: removed the file from the PR

File: rskj-core/src/main/resources/rskj.conf (hunk 2 above)
Comment (oscarguindzberg) [inline rskj-core/src/main/resources/rskj.conf]: removed the file from the PR, sorry

**Comment by oscarguindzberg** [comment]
@aeidelman 
1)  I just commited a change so @author is not mixed with "created by"
2) It is very common classes have several authors. See this example in the jdk source code
http://hg.openjdk.java.net/jdk7/jdk7/jdk/file/00cd9dc3c2b5/src/share/classes/java/util/ArrayList.java

**Pull request merged**